src/type-translator.ts

```diff
@@ -27,17 +27,27 @@ function isClosureProvidedType(symbol: ts.Symbol): boolean {
 
 export function typeToDebugString(type: ts.Type): string {
   const basicTypes: ts.TypeFlags[] = [
-    ts.TypeFlags.Any,           ts.TypeFlags.String,
-    ts.TypeFlags.Number,        ts.TypeFlags.Boolean,
-    ts.TypeFlags.Void,          ts.TypeFlags.Undefined,
-    ts.TypeFlags.Null,          ts.TypeFlags.Enum,
-    ts.TypeFlags.StringLiteral, ts.TypeFlags.TypeParameter,
-    ts.TypeFlags.Class,         ts.TypeFlags.Interface,
-    ts.TypeFlags.Reference,     ts.TypeFlags.Tuple,
-    ts.TypeFlags.Union,         ts.TypeFlags.Intersection,
-    ts.TypeFlags.Anonymous,     ts.TypeFlags.Instantiated,
-    ts.TypeFlags.ObjectLiteral, ts.TypeFlags.ESSymbol,
```

Comment:
I just removed `ObjectLiteral` because that's not marked as `@internal` - I don't know what kind of implications this has on the rest of the code - I couldn't find a good replacement for this type in typescript.d.ts. please advice if this is the right change.